notebooks/official/training/multi_node_ddp_gloo_vertex_training_with_custom_container.ipynb

```diff
@@ -113,11 +115,8 @@
         "### Costs\n",
         "\n",
         "This tutorial uses billable components of Google Cloud:\n",
-        "\n",
         "* Vertex AI\n",
-        "\n",
         "* Cloud Storage\n",
-        "\n",
         "* Vertex AI TensorBoard (Note that Vertex AI TensorBoard charges a monthly fee of $300 per unique active user. Active users are measured through the Vertex AI TensorBoard UI. You also pay for Google Cloud resources you use with Vertex AI TensorBoard, such as TensorBoard logs stored in Cloud Storage. <a href='https://cloud.google.com/vertex-ai/pricing#tensorboard'>Check the link for latest prices.</a>)\n",
```

Comment:
this is not true, anymore: "Vertex AI TensorBoard (Note that Vertex AI TensorBoard charges a monthly fee of $300 per unique active user."